integration

```diff
@@ -629,6 +629,7 @@
   "GyroGearl00se/ha_froeling_lambdatronic_modbus",
   "h4de5/home-assistant-toshiba_ac",
   "h4de5/home-assistant-vimar",
+  "ha-termoweb/ha-termoweb",
```

Comment:
This is from an organisation with no public members, meaning there is no one to contact.
https://github.com/orgs/ha-termoweb/people